### ISSUE TITLE
carla-osc-gui assertion failure

### ISSUE THREAD

**Issue by jkoderu-git** [falkTX/Carla#1275]
`carla-osc-gui project.carxp` starts the enigne and connects to it showing Pulseaudio sinks in patchbay but project is not loaded.

Logs:
```
OSC backend registered to osc.tcp://localhost.localdomain:14826/ctrl
CarlaEngineOsc::sendCallback(29:ENGINE_CALLBACK_ENGINE_STARTED, 0, 1, 2, 1024, 96000.000000, "JACK")
Carla assertion failure: "fPatchbay != nullptr" in file CarlaEngineGraph.cpp, line 2785
CarlaEngineOsc::sendCallback(20:ENGINE_CALLBACK_PATCHBAY_CLIENT_ADDED, 1, 3, 65535, 0, 0.000000, "Carla-01")
```

**Comment by falkTX** [comment]
That is expected. Carla-Control CLI arguments set the network address, and not a project to load.
You need to do that after the GUI is open.

**Comment by jkoderu-git** [comment]
The help displays that the project is loaded automatically

>Usage: carla-osc-gui [OPTION]... [FILE|URL]

>where FILE can be a Carla project or preset file to be loaded, or URL if using Carla-Control

**Comment by falkTX** [comment]
Fixed in e211b3afc43d87df2a71707e57c576e59ecc211c
carla-osc-gui can now take a file argument, but only for the first time you run it.

I cleared up possible confusions in the printed help text for it
